js/blocks-image-customization.js

```diff
@@ -0,0 +1,55 @@
+const { createHigherOrderComponent } = wp.compose;
+
+var largo_core_image_block_add_media_credit = createHigherOrderComponent( function ( BlockListBlock ) {
+
+  return function ( props ) {
+
+    if( props.name === 'core/image' ){
+
+        var attachment_id = props.attributes.id;
+        var media = new wp.api.models.Media( { id: attachment_id } );
+
+        var media_response = media.fetch().then( function( media ) {
+
+            // the original instance of the media caption
+            var attachment_caption = props.attributes.caption;
+
+            if( media.media_credit && media.media_credit._media_credit ){
+
+                // our default media credit, if it exists 
+                var media_credit = '<p class="wp-media-credit">' + media.media_credit._media_credit + '</p>';
+
+                // if our media credit has a url, include that also
```

Comment:
This section tests if it has a media credit org, not if it has a URL.

This PR does not, presently, include a function for including the URL in the media credit.

Reply:
@benlk I wasn't sure about the URL since it was set as a variable but wasn't used in the classic editor way here:
https://github.com/INN/largo/blob/39a0fc4ca34a118ae6ae870dd591029f5a61fdb1/lib/navis-media-credit/navis-media-credit.php#L248-L250

Should the URL wrap the credit or organization name?

Reply:
Odd that it isn't used in the main media credit instance.

It's used for the "hero" featured image on posts, where the link wraps the credit and org name together: https://github.com/INN/largo/blob/72cc8433a1be1de4f22789e1ec89496d54a653d9/partials/hero-featured-image.php#L5-L7